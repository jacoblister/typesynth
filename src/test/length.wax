(func lengthTest
  (print "lengthTest:")
  ; (let vector (vec 3 int) (alloc (vec 3 int) 1 2))
  ; (print (# vector))
  (let array (arr int) (alloc (arr int) 1 2 3))
  (print (# array))
  (let string str "four")
  (print (# string))
  (let allocstring str (alloc str "5five"))
  (print (# allocstring))
)

(func main (result int)
  (call lengthTest)
  (return 0)
)