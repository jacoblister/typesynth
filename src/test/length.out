lengthTest:
3
4
5
